src/transformers/models/siglip/convert_siglip_to_hf.py

```diff
@@ -266,9 +273,9 @@ def convert_siglip_checkpoint(model_name, pytorch_dump_folder_path, verify_logit
 
     # get vocab file
     if "i18n" in model_name:
-        vocab_file = "/Users/nielsrogge/Documents/SigLIP/multilingual_vocab/sentencepiece.model"
+        vocab_file = "/Users/mervenoyan/Desktop/siglips/mc4/sentencepiece.model"
```

Comment:
We could host these on the hub rather than having local paths so that everyone can run it

Reply:
Sure I'll upload them all.

Reply:
Should we have them in same repo as the HF checkpoints? @NielsRogge

Reply:
No, they can be uploaded to a separate repo